### PR TITLE
completed welcome, login, and signup page

### PR DESCRIPTION
**Pull request by jut008**
lmk what text to put into terms of service! theres dummy text for now

### CODE REVIEW

**Review by Shuzhengz — CHANGES_REQUESTED** [review]
probably add newlines at the end of files

look out for warnings like this:

image

**Comment by Shuzhengz** [comment]
> lmk what text to put into terms of service! theres dummy text for now

"Pls giv money"

File: client/src/components/pages/welcome.tsx [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+            <h1>Welcome to Sleep Buddies</h1>
             <div className="welcome_page-div">
-                <Login />
-                <Signup />
+                <Link to="/login">
+                    <button>Login</button>
+                </Link>
+                <Link to="/signup">
+                    <button>Sign Up</button>
+                </Link>
```
Comment (Shuzhengz) [inline client/src/components/pages/welcome.tsx:14]: might wanna use elements for the buttons

Comment (Shuzhengz) [reply client/src/components/pages/welcome.tsx:14]: it seems like yall already created them in `src/components/elements`

**Comment by jut008** [comment]
fixed the issues! someone please take a look

**Review by JingtingL — DISMISSED** [review]
Looks good to me

**Review by jut008 — COMMENTED** [review]
fixed

File: client/src/components/pages/welcome.tsx [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import React from "react";
 import Login from "../elements/Login";
 import Signup from "../elements/Signup";
-const welcome = () => {
+import "../../css/welcome.css";
```
Comment (Shuzhengz) [inline client/src/components/pages/welcome.tsx:4]: file doesn't exist

**Review by Shuzhengz — CHANGES_REQUESTED** [review]
the app errors out

File: client/src/components/pages/welcome.tsx (hunk 2 above)
Comment (jut008) [reply client/src/components/pages/welcome.tsx:4]: forgot to include it should be there now

Comment (Shuzhengz) [reply client/src/components/pages/welcome.tsx:4]: ah ok lemme try again

**Review by Shuzhengz — DISMISSED** [review]
lgtm

**Review by Shuzhengz — DISMISSED** [review]
lgtm

**Review by JingtingL — APPROVED** [review]
Looks good to me

**Review by Shuzhengz — APPROVED** [review]

**Pull request merged**